pkg/crd/markers/crd.go

```diff
@@ -31,6 +31,7 @@ var CRDMarkers = []*markers.Definition{
 	markers.Must(markers.MakeDefinition("kubebuilder:subresource:scale", markers.DescribesType, SubresourceScale{})),
 	markers.Must(markers.MakeDefinition("kubebuilder:printcolumn", markers.DescribesType, PrintColumn{})),
 	markers.Must(markers.MakeDefinition("kubebuilder:resource", markers.DescribesType, Resource{})),
+	markers.Must(markers.MakeDefinition("kubebuilder:conversion:storage", markers.DescribesType, StorageVersion{})),
```

Comment:
Is `kubebuilder:crd:storage` better?

Reply:
Users need to specify which version is storage version, even when they don't use conversion.

Reply:
agreed.